src/components/footer/Footer.css

```diff
@@ -0,0 +1,13 @@
+.footer {
+  width: 100%;
+  background: #ef9364;
```

Comment:
I think black background and white text in .footer class to match the rest of the app would look good.

Reply:
I followed design: https://www.figma.com/file/F7sNzfbrDwaWdslg5R5ZCS/PRACT-Flashcard-App?type=design&node-id=0-88&mode=design&t=Y5C7OGzA6Jv8GSNO-0